StacksOfPlates.py

```diff
@@ -0,0 +1,48 @@
+class Stack:
+    def __init__(self, cap):
+        self.stack = []
+        self.cap = cap
+
+    def isFull(self):
```

Comment:
The naming convention for functions in Python is the same as the naming convention for variables.
```suggestion
    def is_full(self):
```